test/sim/abilities/lightningrod.js

```diff
@@ -82,10 +82,23 @@ describe('Lightning Rod', function () {
 			{species: 'Pichu', ability: 'static', moves: ['thunderbolt']},
 			{species: 'Pichu', ability: 'static', moves: ['thunderbolt']},
 		]});
-		const [rodCopied, rodStarts] = battle.p1.active;
+		let [rodCopied, rodStarts] = battle.p1.active;
 		battle.makeChoices('move roleplay -2, move sleeptalk', 'move thunderbolt 1, move thunderbolt 2');
 		assert.statStage(rodCopied, 'spa', 0);
 		assert.statStage(rodStarts, 'spa', 2);
+
+		battle = common.createBattle({gameType: 'doubles'}, [[
+			{species: 'Togedemaru', moves: ['zingzap']},
+			{species: 'Ditto', ability: 'imposter', moves: ['transform']},
+		], [
+			{species: 'Manectric', ability: 'lightningrod', moves: ['sleeptalk']},
+			{species: 'Dratini', moves: ['thunderbolt']},
+		]]);
+		rodCopied = battle.p1.active[1]; // Ditto
+		rodStarts = battle.p2.active[0]; // Manectric
+		battle.makeChoices('move zingzap 1, move sleeptalk', 'move sleeptalk, move thunderbolt 2');
```

Comment:
Would it be better to have Zing Zap's target be Dratini (2) and Thunderbolt's target be Togedemaru (1)? Either way it _should_ redirect to Manectric, but then it's explicit that it has to redirect to one of the two.

I would double check to make sure this also has the same behavior with a faster Imposter and a slower LightningRod (e.g. make the Manectric a Pincurchin or smth). Probably doesn't need a second test written, but check if tossing a Pincurchin still passes locally just to make sure.

Reply:
> That's what I was going for. I thought `1` was on the user's left, which would be the second member of the team. Is it not?

1 is the left slot from the _opponent's_ perspective. 2 is the right slot from the opponent's perspective. If you're like me, I get it confused a lot because I think 0 is first and 1 is second like an array. Dealing with perspective is annoying, but yeah, target is from opponent's left to right. So I would still update the targets to be slightly more clear.

> The imposter would just copy the Pincurchin and end up just as slow, right? I'll give the Lightning Rod pokemon Speed Swap and have it swap with Dratini for a second round to make sure the Imposter takes the redirects in that case.

Speed Swap doesn't affect the order at all [according to Marty](https://discord.com/channels/506642369068662784/506643485731061780/515198772569964555), so I wouldn't include it in the actual test. In any case, I pulled the changes locally and it still behaves correctly with a Pincurchin in place of Manectric. Should be good there!